DTS-STN/dts-rmp
# changed dev script to nuxt dev

# Description

We were not developing properly for nuxt. Nuxt documentation states we should be using `nuxt dev` for development. We have been using nodemon. Using nodemon made it so that in development we were not using the application like a nuxt application and thus problems with our serverMidldleware/axios config were not seen (the app cannot fetch engagements in this case). After deploying the application to production with its first api call, we found that the middleware/axios was not properly configured. After testing the application with "nuxt dev", the problems were reproducible locally. Running the application in production mode locally will also reveal the same problem.

## Test Instructions

1. npm run mongodb
1. npm run seed
1. npm run dev
1. go to page and see that no engagement data is received

OR

1. npm run mongodb
1. npm run build
1. npm run start
1. go to page and see that no engagement data is received
